modules/label.rb

```diff
@@ -13,4 +13,8 @@ def add_items(item)
     @items << item
     item.label = self
   end
+
+  def to_json(*_args)
+    { 'id' => @id, 'title' => @title, 'color' => @color }
```

Comment:
The same thing here as to using hashes as described above, it will be ok if you can edit this.